spec/features/post_index_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'rails_helper'
+
+RSpec.describe 'post_index', type: :feature do
+  user = User.create(name: 'Bill', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Chef from Ireland.',
+                     posts_counter: 0)
+  post = Post.create(title: 'Pizza Recipe', text: 'Recipe for a tasty pepporoni pizza', comments_counter: 0,
+                     likes_counter: 0, user_id: user.id)
+
+  before(:each) do
+    visit user_posts_path(user.id)
+  end
+
+  it 'shows the username of the the user' do
+    expect(page).to have_content(user.name)
+  end
+
+  it 'shows the profile photo of user' do
+    expect(page).to have_selector("img[src='#{user.photo}']")
+  end
+
+  it 'shows the number of post of each user' do
+    expect(page).to have_content('Number of posts: 1')
+  end
+
+  it 'shows the post\'s title' do
+    expect(page).to have_content(post.title)
+  end
+
+  it 'shows the post\'s body' do
+    expect(page).to have_content(post.text)
+  end
+
+  it 'shows the post\'s comments' do
+    expect(page).to have_content('Testing comment')
+  end
```

Comment:
- [ ] One of your tests is failing. You did not create a comment with the text: "Testing comment", as your test assertion expects. You can correct this by creating a comment just as you have created a post and user above.

  ```
  comment = Comment.create(text: "Testing comment", post_id: post.id, user_id: user.id)
  ```
  ![post-t](https://user-images.githubusercontent.com/54848363/187913510-c4e6b272-5b98-4838-bf39-cbf5d667ae05.jpg)